packages/v4/src/content/design-guidelines/components/empty-state/empty-state.md

```diff
@@ -3,208 +3,393 @@ id: Empty state
+
+## Elements
+This is an example of a large, full-page empty state with all empty state elements. Other empty state variations contain the same elements and differ only in spacing and text sizes. 
+
+<img src="./img/empty-state.png" alt="large empty state example with all elements" width="1500px"/>
+
+1. **Icon**: The icon presents a visual component for easier recognition of the empty state’s message.
+2. **Heading**: The title provides a concise description of the empty state’s purpose.
+3. **Body**: The body provides additional information to help the user understand why the space is empty, what they can do to move forward, and how they can benefit from taking the next step.
+4. **Primary button**: The primary button enables the user to take either the next step or a more general action, such as returning to the home page. 
+5. **Secondary buttons**: Secondary buttons enable the user to take alternative actions, such as accessing additional resources. 
+
+**Note**: Don't include toolbars or filters in a full-page empty state because the user doesn't have any items to search or filter. 
 
 ## Usage 
-PatternFly offers four empty state variations; an extra small empty state, a small empty state, a large empty state, and an extra large empty state. 
-* The **extra small empty state** should be used inside cards, or when there are space constraints, as seen in the second [no access](#user-not-entitled-or-does-not-have-access) use case example. 
-* The **small empty state** should be used inside tables, wizards, modals, or cards, where the empty state is "inside a container," as seen in the [no data to show](#no-data-to-show) use case example. 
-* The **large empty state** should be used in a full-page empty state, as seen in the [addition or creation](#addition-or-creation) use case example. 
-* The **extra large empty state** variation should only be used in special instances, specifically for [first-use or get started](#first-use-or-get-started) states, as well as for full-page [congratulatory](#success) states, seen in those use case examples.
+PatternFly offers four empty state variations: 
 
-## Elements
-Below is an example of a large empty state inside a full-page, outlining all the elements inside an empty state. The other empty state variations are made up of the same elements, and only differ in their spacing and text sizes.
+- **Extra small empty state**: Use an extra small empty state inside cards or when there are space constraints. 
+- **Small empty state**: Use a small empty state inside tables, wizards, or modals. 
+- **Large empty state**: Use a large empty state as a full page. 
+- **Extra large empty state**: Use an extra large empty state for special instances only, such as full-page success empty states or getting started empty states. 
 
-<img src="./img/empty-state.png" alt="empty state example" width="990"/> 
+There are several common uses for empty states:  
```

Comment:
would be useful to link these to their corresponding section so a user can directly jump to the use case they want instead of having to scroll!

Reply:
Good idea! Do you know how I can insert those jump links?

Reply:
[text you want users to see](#text-that-matches-header-section-separated-by-dashes)
Make sure the text that appears after the # is **exactly** the same as the section header you're linking to, with each space being replaced by a dash.  Also make sure it's all in lower case!